elastic/kibana
# [Discover] Add data view changed warning after alert rule created

## Summary

Closes #134232

This PR adds extra waring in case the user updated data view after alert rule creation/updating.

### Test  
- Create an alert off a query that uses a certain field
- Add field as a field filter in a Data View
- Follow the link to see results triggered the alert. 
- Filtered field will not be shown. Instead user should see additional warning.

![455C6C75-7467-41EE-8012-ACA8AF03546B_4_5005_c](https://user-images.githubusercontent.com/39378793/174324159-d7500d22-d79b-47dc-960f-0ab16ea9ded4.jpeg)





### Checklist

Delete any items that are not applicable to this PR.
